src/grapple.c

```diff
@@ -11,6 +11,35 @@
 
 void SpawnBlood(vec3_t dest, float damage);
 
+float StartSpeed(vec3_t velocity)
+{
+	return sqrt(
+		velocity[0] * velocity[0] +
+		velocity[1] * velocity[1] +
+		velocity[2] * velocity[2]
+	);
+}
+
+float IncrementSpeed(float speed, float increment)
+{
+	float newSpeed = 800; 
+	if (speed < (800 - increment))
```

Comment:
code style:
please tag the function with some newlines:
- after the variable definitions/declarations
- after a block (closing bracket of an if statement)